deploy/charts/cert-manager/templates/rbac.yaml

```diff
@@ -507,7 +507,7 @@ subjects:
 ---
 
 # Permission to:
-# - Update and sign CertificatSigningeRequests referencing cert-manager.io Issuers and ClusterIssuers
+# - Update and sign CertificateSigningRequests referencing cert-manager.io Issuers and ClusterIssuers
```

Comment:
Typically I'd make two commits for something like this, but it looks like `Signing` was pasted into `Certificate`, and thus it wasn't two misspelled words, just one corrupted word.